docs/logs-events/log-restoration/index.md

```diff
@@ -0,0 +1,83 @@
+    ![Restored logs list](restored-logs-list.png)
+1. Click ![Create new icon](/docs/one-eye/headless/icon-create-new.png). The Log Restoration Wizard opens.
+    ![Log Restoration Wizard](log-restoration-output.png)
+1. Select the output from which you want to restore logs, then click **Next**.
+1. Select the cluster and the time interval for which you want to restore logs.
+    ![Log Restoration Wizard](log-restoration-cluster.png)
+1. To specify the details of the logs to be restored on the UI, select **Simple Builder**. Alternatively, you can select **Advanced Builder**, and paste a [JSON that describes the logs to restore](#builder-json).
+1. To restore the logs of only specific namespaces, pod, and containers, search for the name of the respective resource, and select the ones you need. If you want to restore the logs of every resource, select **Any**. Note that searching for the resources returns only exact matches, unless you include the * wildcard.
+    ![Filter logs to restore](log-restoration-namespace.png)
+1. Set a prefix for the Elastic index. In addition to using it as a prefix for the Elastic index, the Elastic resources are named after this parameter.
+    ![Set Elastic parameters](log-restoration-configuration.png)
+1. Set the namespace where the log restoration job will run. The Elastic instance is also deployed into this namespace.
+1. Before restoring the logs starts, review the settings of the process. If everything is as you intended, click **Next**.
+    ![Log Restoration overview](log-restoration-overview.png)
+1. One Eye starts restoring the logs. Depending on the amount of logs to restore, this can take a long time.
+    ![Log Restoration overview](log-restoration-progress.png)
+1. After the job is successfully completed, you can [access the restored logs](#access-logs).
+
+## Access restored logs {#access-logs}
+
+To tail your logs, or the logs that match a specific query in real-time, complete the following steps.
+
+1. [Open the One Eye web interface]({{< relref "/docs/one-eye/quickstart/_index.md#display-the-logging-flow" >}}).
+1. Select **MENU > LOG RESTORATION**. This page the log restoration jobs and their details.
+    ![Restored logs list](restored-logs-list.png)
+1. Click the ID of a set of logs. The access details of the logs is displayed.
+    ![Restored logs details](restored-logs-details.png)
+1. Click the URL to open the Elasticsearch service where the logs are available, and use the username and password to access them (click the eye icon to display the password).
+
+## Delete the restored logs
+
+To delete the restored logs and/or the Elastic instance from the One Eye cluster, you must delete the related resources manually. The Elastic resources are named after the **Index prefix** parameter of the restored logs, and run in the namespace specified in the configuration of the log restoration.
+<!-- FIXME operator/cli mode link or description -->
```

Comment:
Are you looking for a CLI command to delete the Elastic instance? If so: `kubectl delete elasticsearch -n <namespace> <index prefix>`